### PR TITLE
docs: fix typos in configuration.md

### PR DESCRIPTION
**Pull request by richdouglasevans**

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -200,10 +200,10 @@ Default: `['clear-text', 'progress', 'html']`<br />
 Command line: `--reporters clear-text,progress,dots,dashboard,html,json`<br />
 Config file: `"reporters": ["clear-text", "progress", "dots", "dashboard", "html", "json"]`
 
-With `reporters`, you can set the reporters for stryker to use.
+With `reporters`, you can set the reporters for Stryker to use.
```
Comment (richdouglasevans) [inline docs/configuration.md:203]: Consistent capitalisation of the product name.

File: docs/configuration.md [hunk 2]
```diff
@@ -200,10 +200,10 @@ Default: `['clear-text', 'progress', 'html']`<br />
...
 These reporters can be used out of the box: `html`, `json`, `progress`, `clear-text`, `dots`, `dashboard` and `event-recorder`.
 By default, `clear-text`, `progress`, `html` are active if no reporters are configured. See [reporter plugins](./plugins.md#reporters)
-for a Fdescription on each reporter.
+for a fuller description of each reporter.
```
Comment (richdouglasevans) [inline docs/configuration.md:206]: I guessed at what this was meant to be; reads okay to me now.

Comment (kmdrGroch) [reply docs/configuration.md:206]: maybe just "full" description? No need for "fuller" IMO :D

Comment (richdouglasevans) [reply docs/configuration.md]: Done.

File: docs/configuration.md (hunk 1 above)
Comment (nicojs) [reply docs/configuration.md:203]: Yeah, true! Thanks!

**Comment by nicojs** [comment]
Thanks for the PR! I guess the "edit" button underneath the pages on the website is working? 😄

**Comment by richdouglasevans** [comment]
> Thanks for the PR!

Thanks for your continued work on Stryker! 🙇 

> I guess the "edit" button underneath the pages on the website is working? 😄

Oh! 😮 I just forked the repo in the usual way from my shell: totally missed the "edit" button.

**Pull request merged**